grpc/send_incoming_call_notification.go

```diff
@@ -0,0 +1,54 @@
+package grpc
+
+import (
+	"bytes"
+	"context"
+	"encoding/binary"
+	"log"
+
+	pb "github.com/PretendoNetwork/grpc-go/friends"
+	nex "github.com/PretendoNetwork/nex-go"
+	friends_wiiu_types "github.com/PretendoNetwork/nex-protocols-go/friends-wiiu/types"
+	nintendo_notifications "github.com/PretendoNetwork/nex-protocols-go/nintendo-notifications"
+	nintendo_notifications_types "github.com/PretendoNetwork/nex-protocols-go/nintendo-notifications/types"
+	"github.com/PretendoNetwork/wiiu-chat-secure/globals"
+	"google.golang.org/grpc/metadata"
+)
+
+func SendIncomingCallNotification(caller uint32, target uint32) {
+	ctx := metadata.NewOutgoingContext(context.Background(), globals.GRPCFriendsCommonMetadata)
+
+	presence := friends_wiiu_types.NewNintendoPresenceV2()
+
+	presence.ChangedFlags = 0x1FF
```

Comment:
Do we know what flags these are? I'm wondering if we should build some kind of flag building system as well for these

Reply:
These flags are unknown to me (also brought up in friends 3ds).
If anyone *does* know them a flag system would probably be handy.

Reply:
Looks like some flags _are_ known actually https://nintendo-wiki.pretendo.network/docs/nex/protocols/friends-wiiu#changed-flags

Reply:
This actually has no effect here, so I've removed it.